instructions.md

```diff
@@ -0,0 +1,41 @@
+# Welcome to NodeSchool!
+
+Wi-Fi: Curiosity Media - Guest
+
+## Pair programming
+
+We consider working in pairs one of the most important skills in programming.
+It helps clarify your thinking and keeps you from getting stuck on things
+you don't understand.
+
+###### Good pairing
+- Only use one laptop
+- Switch who is using the keyboard at least each time you move on to a new
+  exercise
+- Think outloud
+
+## Getting Started
+
+- Go to `www.nodejs.org` and download either v4 or v5.
+- Get a text editor if you don't already have one. The following are all free
+  and recommended:
+  * Atom
+  * Sublime Text 2
+  * Brackets
+  * Textmate
+- Go to `www.nodeschool.io` to look at the workshop instructions
+- We recommend the following workshops:
+
+New to javascript            | javascripting
+New to Node.js               | learnyounode
+Some experience with Node.js | stream-adventure
+
+## Tips for doing the workshops
+
+- Read all of the instructions! It's easy to miss parts
+- If you want to know what's going on with your code, use
+`console.log('myData', myData)`. It will first print the string `myData` to
+the screen and then the value of the variable `myData`
+- Ask questions! Especially when you get to working with callbacks or modules.
+Raise your hand at any time or get one of the mentor's attention. We're glad to
+help :)
```

Comment:
Call out the name-of-workshop run as a way to troubleshoot, and name-of-workshop verify to confirm DO NOT use node my-program.js my made up params